### PR TITLE
Syntax highlighting

### LINKED ISSUE #10: Color
man pages on a typical linux terminal are quite colorful. Sometimes terminals display bold and italic in two different colors, others with underline, etc. This is useful for quickly eyeballing keywords.

Could you please implement something like that here too? Either bold and italic fonts or different color fonts.

### PR DESCRIPTION
**Pull request by meronz**
This PR builds upon the work on PR https://github.com/meronz/vscode.manpages/pull/24 by @MishaKlopukh.

### CODE REVIEW

**Comment by meronz** [comment]
@MishaKlopukh is it okay for you if we merge this branch? I made some additional work on top of you branch, like adding a setting, reverting some changes that made the extension not work, and increasing the release number.

I would also like to squash it to avoid polluting the history too much.

**Comment by GitMensch** [comment]
I'd be very interested in that. @MishaKlopukh any comments from your side?

**Comment by MishaKlopukh** [comment]
Looks good to me.

**Comment by meronz** [comment]
okay i'll merge this later this week!

File: src/manpageContentProvider.ts [hunk 1]
```diff
@@ -181,4 +185,7 @@ export async function openManPage(input: string, inActiveColumn: boolean): Promi
         viewColumn: inActiveColumn ? vscode.ViewColumn.Active : vscode.ViewColumn.Beside
     });
     textDocument.options.lineNumbers = 0; // off
+
+    const enableSyntaxHighlighting = (vscode.workspace.getConfiguration('manpages').get('enableSyntaxHighlighting', true) as boolean);
+    if(enableSyntaxHighlighting) vscode.languages.setTextDocumentLanguage(doc, 'manpage');
```
Comment (GitMensch) [inline src/manpageContentProvider.ts:190]: That does work quite well and the syntax is as expected - but it seems that if we follow links to open a manpage this has to be done there as well (as of version 0.2.0 links work, but they are highlighted as "text-only", so a user has to manually adjust the syntax).

Ideally, this option can be dropped here and instead there a language detector be implemented - see #39.
As an alternative (or shortcut for now) - at the place where we open links do the same as those two lines.

**Pull request merged**